root/static/scripts/work/edit.js

```diff
@@ -66,16 +79,20 @@ const workLanguageOptions: MaybeGroupedOptionsT = {
   options: scriptArgs.workLanguageOptions,
 };
 
-const store = createStore(function (state: WorkForm = form, action) {
+// eslint-disable-next-line func-call-spacing
```

Comment:
Is this to avoid some weird indenting catch-22?

Reply:
I think it's actually an eslint bug.  I can add a comment above.